src/admin/body/users.js

```diff
@@ -0,0 +1,34 @@
+export const users = [{
```

Comment:
Let's put that curly brace on a new line.

Reply:
![image](https://user-images.githubusercontent.com/23569938/125927607-535c8558-aca2-425f-9100-94e17a9d68eb.png)
Also, this image looks pretty small on mobile. Let's hide the id and increase its size on small screens.